critiquebrainz/frontend/templates/macros.html

```diff
@@ -51,3 +51,11 @@
     </script>
   {% endif %}
 {% endmacro %}
+
+{% macro review_credit(review) %}
+{{ _(
+  'Event review by %(user)s' if review.entity_type == 'event' else 'Music review by %(user)s' if review.entity_type == 'release_group' else 'Review by %(user)s',
```

Comment:
@gentlecat From a0b68ac92's commit message:

> http://i.imgur.com/UTbDvxj.png
> Previously, the review byline would sometimes get cut off. It has been
shortened.

I think that experienced users will not be confused by the `Music` because there is only one musical entity type that reviews apply too. As for new users, they would be as if not less confused by `Music` than by `Release Group`, not being familiar with the latter term.

Yet another alternative might be to move the entity type indication to the _end_ of the review credits; however, this too results in ugly chopping off.